Documentation/contributing/testing/e2e.rst

```diff
@@ -212,6 +217,35 @@ has been resolved):
 
 Finally, you can use the instructions from :ref:`the previous chapter<test_cilium_on_lvh>` to run and to test Cilium.
 
+Network performance test
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Cilium also provides `cilium-cli connectivity perf
+<https://github.com/cilium/cilium-cli/?tab=readme-ov-file#network-performance-test>`_
```

Comment:
```suggestion
<https://github.com/cilium/cilium-cli/blob/756ae5072a7281e2b6b33b10da3ab890cb2fa240/README.md#network-performance-test>`__
```
I'd suggest using a permalink here, and please use an anonymous hyperlink (with two underscores, see [docs style guide](https://docs.cilium.io/en/stable/contributing/docs/docsstyle/#links)).